src/util/WindowsHelper.cpp

```diff
@@ -83,6 +88,55 @@ void setRegisteredForStartup(bool isRegistered)
     }
 }
 
+QString getAssociatedCommand(AssociationQueryType queryType, LPCWSTR query)
+{
+    static HINSTANCE shlwapi = LoadLibrary(L"shlwapi");
+    static auto assocQueryString = AssocQueryString_(
+        (shlwapi == NULL) ? NULL
+                          : GetProcAddress(shlwapi, "AssocQueryStringW"));
```

Comment:
would be more readable to return early when LoadLibrary fails instead of ternary use here